src/styles/elements/c-footer.scss

```diff
@@ -9,6 +9,10 @@
     color: $light-01; // IE
     text-decoration: underline;
   }
+
+  &.active { // IE
+    text-decoration: underline;

```

Comment:
The rules should be implemented on desktop as well, either change %nav-link-desktop to mixin or add it to ::slotted(a.active)

Reply:
Make sure to add all different active state possibilities (class & attributes). Should add it to `:slotted(a:active),
::slotted(a.active),
::slotted(a[active]:not([active="false"]))`

Reply:
Added it for slotted,

Please check both for social media active and regular link active, since they are two different situations